components/automate-chef-io/Makefile

```diff
@@ -11,6 +11,15 @@ clean:
 sync: themes/chef
 	pushd themes/chef && git fetch && git reset --hard origin/master && popd
 
+SWAGGER_DIR=data/docs/api_chef_automate
+
+clean_swagger_files:
+	rm -rf $(SWAGGER_DIR)/*
+
+sync_swagger_files: clean_swagger_files
+	pushd ../../api/external && find . -name '*.swagger.json' | cpio -p -dumv ../../components/automate-chef-io/$(SWAGGER_DIR) && popd
+	pushd ../automate-gateway/api/ && find . -name '*.swagger.json' | cpio -p -dumv ../../automate-chef-io/$(SWAGGER_DIR) && popd
```

Comment:
This was trickier than it seemed at first because of the need to work across osx and linux. Busybox's (that's what the hab studio uses, right?) cpio supports all of these cli flags so hopefully this will just work: https://busybox.net/downloads/BusyBox.html

Reply:
`cp` won't copy the directory structure, will it?

Reply:
Oh! Wasn't aware of that detail 😓 it doesn't. Ignore me 😃

Reply:
One of the issues solved in this PR is that all of the swagger files were copied into one directory, and there are two `users.swagger.json`, for example.

Reply:
This solution is one of the few I could come up with that didn't get into `sed`ing the paths of files, etc, that got pretty gross quickly.